app/controllers/collections_controller.rb

```diff
@@ -85,6 +85,11 @@ def edit_link
     }
   end
 
+  def admin
+    @collection = Collection.find(params[:id])
+    authorize! @collection
+  end
```

Comment:
This is the new endpoint called when the browser gets a turbo frame pointing at this URL.